src/OpenConvenanten/Taxonomy/TaxonomyController.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Yard\OpenConvenanten\Taxonomy;
+
+class TaxonomyController
+{
+    /**
+     * Add 'show on' additional explanation.
+     */
+    public function addShowOnExplanation(string $taxonomy): void
+    {
+        if ('openconvenanten-show-on' !== $taxonomy) {
+            return;
+        }
+
+        echo '<div class="form-field">
```

Comment:
esc_html?

Reply:
esc_html() escapes a string so that it is not parsed as HTML & 'But there's no point escaping HTML that you have hard-coded into the template or function, since it's predictable.'